src/main/kotlin/io/github/enyason/predictions/Extensions.kt

```diff
@@ -0,0 +1,12 @@
+package io.github.enyason.predictions
+
+import java.net.URI
+
+fun String?.getCursor(): String {
```

Comment:
cursor should be nullable. If you can't retrieve it, return null instead of an empty string.

the code will read clearer if you just return:
`URI(this).query?.split("&") ?.firstOrNull { it.contains("cursor") }?.split("=")?.get(1)`

Reply:
I agree. Thanks